server/api/initial_data.py

```diff
@@ -245,6 +246,8 @@ def _perform_data_migrations(db_session: sqlalchemy.orm.Session):
                 _perform_version_6_data_migrations(db, db_session)
             if current_data_version < 7:
                 _perform_version_7_data_migrations(db, db_session)
+            if current_data_version < 8:
+                _perform_version_8_data_migrations(db, db_session)
```

Comment:
aren't you supposed to bump the latest data version then?

Reply:
@liranbg already did it and it's merged #6554

Reply:
And in #6556 for 1.7.x